libtracepoint-control-cpp/src/TracepointSession.cpp

```diff
@@ -1266,11 +1455,12 @@ TracepointSession::ParseSample(
 
 Done:
 
-    enumEventInfo.event_desc = infoEventDesc;
-    enumEventInfo.session_info = &m_sessionInfo;
-    enumEventInfo.id = infoId;
-    enumEventInfo.raw_data = infoRawData;
-    enumEventInfo.raw_data_size = infoRawDataSize;
+    m_enumEventInfo.event_desc = infoEventDesc;
+    m_enumEventInfo.session_info = &m_sessionInfo;
+    m_enumEventInfo.header = infoHeader;
+    m_enumEventInfo.id = infoId;
+    m_enumEventInfo.raw_data = infoRawData;
+    m_enumEventInfo.raw_data_size = infoRawDataSize;
```

Comment:
Note: Nothing major changed here, just a bit of rearrangement of how I fill-in the eventInfo structure.
- `enumEventInfo` is now always a reference to `m_enumEventInfo`. This was useful a long time ago but is now just an unnecessary layer of abstraction, so use `m_enumEventInfo` directly.
- Add an initialization of `header`.